contributors/devel/security-release-process.md

```diff
@@ -82,14 +112,47 @@ The communication to users should be actionable. They should know when to block
+- The Release Managers will ensure all the binaries are built,
+  publicly available, and functional before the Release Date.
+    - Note: we ship source with our binary releases, so as soon as binary
+      artifacts are publicly available, the fixes (and vulnerabilities) are
+      effectively public.
+  - CI testing of the private patch release branch should provide confidence in
+    the release.
+- The Release Manager will use release tooling (i.e. `anago`) to
+  - Compute the version for the new security patch release; in the previous
+    example, the `release-2.4.7` branch would produce a `v2.4.8` release.
+  - Verify that no public patch release has occurred since the security branch
+    was cut (in this example, make sure that `v2.4.8` has not been tagged
+    publicly).
+  - Perform a normal release build from the private release branch, staging
+    binary artifacts locally (or possibly in a private bucket).
+  - Push the new patch release tag to the private repo.
+  - The tooling should **not** upload any artifacts to public storage buckets or
+    create a draft release on the public GitHub repo.
+- As soon as we are ready to make the fix known publicly (TODO: when?),
+  the Release Manager will use tooling to
+  - Publish results.
+    - Upload binary artifacts to GCS, gcr.io, and GitHub. (TODO: are these
+      prebuilt from the previous step?)
+    - Update the `CHANGELOG.md` in the **public** `master` branch
+    - Create a new release on the **public** kubernetes GitHub
+    - Send email announcing the release
+  - Merge the private patch release branch into the **public** release branch
+    including the fixes and release tag.
+    - In our continuing example, this would merge the private `release-2.4.7`
+      branch into the public `release-2.4` branch, effectively advancing the
+      latter to `2.4.9-beta.0+`. Any changes in the `release-2.4` branch will
+      now ship with `v2.4.9`.
+  - If fixes were necessary to `master`, merge the private `master` branch into
+    the public `master` branch. No release will be built or published for
+    `master`.
+    - TODO: should this be a PR or a push?
 - The Fix Lead will request a CVE from [DWF](https://github.com/distributedweaknessfiling/DWF-Documentation) and include the CVSS and release details.
 - The Fix Lead will email kubernetes-{dev,users,announce,security-announce}@googlegroups.com now that everything is public announcing the new releases, the CVE number, the location of the binaries, and the relevant merged PRs to get wide distribution and user action. As much as possible this email should be actionable and include links how to apply the fix to users environments; this can include links to external distributor documentation.
 - The Fix Lead will remove the Fix Team from the private security repo.
+- The Fix Lead will (possibly?) delete the patch release branch from the private
```

Comment:
the tag holds the pointer to the released sha, I don't see a need to pollute with point release branches